DAILY.md

```diff
@@ -17,8 +17,8 @@ days, if you feel like sharing.
 * _(optional)_ Mention any blockers.
 
 ### Daily Examples (taken from our best devs at hipchat):
-  |   |   |
----|---|---|---
+Dev Name | Daily | Time
+---|---|---
 **Ignacio Mendizabal** | Daily: scale testing of conflux and announce i'm gonna be father. | ``11:03 AM``
 **Iñaki Garay** | daily: finishing choosy image processing robustification and starting an epic bughunting rampage | ``11:33 AM``
 **Marcelo Gornstein** | daily: working with @amilcar on the erlang factory talk and sumodb, also discussing ac improvements for data loading with @brujo and the tt guys, also some code review for jay | ``10:55 AM``
```

Comment:
I found 9 misspelled words in this sentence: 
- |
	- *Suggested Replacements:* , k, p, u, e, z, j, o, t, d, y, i, n, s, c, x, h, m, r, b, w, g, l, q, a, v, f
- tt
	- *Suggested Replacements:* vt, mt, dt, ti, ptt, tn, rt, t, ts, it, tx, gtt, wt, tb, nt, att, ut, tg, et, trt, lt, tyt, tit, ta, st, ct, tv, jt, tgt, tk, qt, at, tnt, tp, ht, tut, tet, tu, te, ot, ft, tst, to, tat, kt, txt, bt, tot, tc, yt, th, pt, tm, gt, tr, tty, tkt
- @brujo
	- *Suggested Replacements:* brujo
- sumodb
- erlang
- @amilcar
- |
	- *Suggested Replacements:* , k, p, u, e, z, j, o, t, d, y, i, n, s, c, x, h, m, r, b, w, g, l, q, a, v, f
- Gornstein**
- **Marcelo